examples/python/helloworld/helloworld_pb2_grpc.py

```diff
@@ -19,7 +19,20 @@ def __init__(self, channel):
                 '/helloworld.Greeter/SayHello',
                 request_serializer=helloworld__pb2.HelloRequest.SerializeToString,
                 response_deserializer=helloworld__pb2.HelloReply.FromString,
-                )
+                _registered_call_handle=channel._get_registered_call_handle(
+                    '/helloworld.Greeter/SayHello'))
+        self.SayHelloStreamReply = channel.unary_stream(
+                '/helloworld.Greeter/SayHelloStreamReply',
+                request_serializer=helloworld__pb2.HelloRequest.SerializeToString,
+                response_deserializer=helloworld__pb2.HelloReply.FromString,
+                _registered_call_handle=channel._get_registered_call_handle(
```

Comment:
Instead of duplicating the method string and requiring creating a semi-private method on `Channel`, could we instead just have a kwarg called `_registered` which takes a `bool` and has a default of `False`? The smaller change to the API surface seems much more appealing.

This is similar to what you've done for the simple stubs versions.